### PR TITLE
Issue#551

### PR DESCRIPTION
**Pull request by himanshuc3**

### CODE REVIEW

**Comment by himanshuc3** [comment]
I've only made one change. Why are 3 files affected here. What could be wrong here @catarak.
I've just changed one line in overlay.scss for this commit.

**Comment by catarak** [comment]
maybe your branch hasn't been rebased on top of the latest master?

**Comment by himanshuc3** [comment]
I think now this could be safely merged.

**Comment by catarak** [comment]
it would be cool if instead of the inputs being hidden, it would shrink to the size of the overlay!

**Comment by himanshuc3** [comment]
You mean like reducing font size and all i.e. adding media queries?

**Comment by catarak** [comment]
no, by making the width of the inputs 100% of the parent element so they don't overflow but rather their width shrinks!

**Comment by himanshuc3** [comment]
oh, you don't want the scrollbars, is it correct?

**Comment by catarak** [comment]
yes exactly!

**Comment by catarak** [comment]
you now have two PR's open for #551... which one do you want to merge in?

**Comment by himanshuc3** [comment]
Had some issue with the git branches. Both PRs are the same, this one just has a newline extra compared to the other. I think you can merge this one and as soon as you do it, I will close the other one too!

**Comment by catarak** [comment]
no worries! git is definitely not easy to use and i struggle with it all of the time 🌈

**Comment by catarak** [comment]
do you think you could try to make this work without media queries? i don't think that you need them. maybe using flexbox would help?

**Comment by catarak** [comment]
i'm gonna go ahead and close the other PR just so i don't get confused.

**Comment by himanshuc3** [comment]
@catarak maybe flexbox could do the trick. I would look into that.

**Comment by himanshuc3** [comment]
Now the media queries are removed. They are only present so that on smaller screens, text is centered that improves accessibility, i guess!

File: client/styles/components/_share.scss [hunk 1]
```diff
@@ -1,7 +1,8 @@
 .share-modal {
   padding: #{20 / $base-font-size}rem;
   padding-top: 0;
-  width: #{500 / $base-font-size}rem;
+  // width: #{500 / $base-font-size}rem;
```
Comment (catarak) [inline client/styles/components/_share.scss]: could you remove this comment?

File: client/styles/components/_copyable-input.scss [hunk 2]
```diff
@@ -50,4 +51,15 @@
     color: getThemifyVariable('button-background-hover-color');
     border-top-color: getThemifyVariable('button-background-hover-color');
   }
+}
+
+/*============================
+== Media queries
+============================*/
+
+@media only screen and (max-width: 775px) {
```
Comment (catarak) [inline client/styles/components/_copyable-input.scss]: could you make this media query into a SASS mixin? and put the width 775 into a variable? https://medium.com/codeartisan/breakpoints-and-media-queries-in-scss-46e8f551e2f2

Comment (himanshuc3) [reply client/styles/components/_copyable-input.scss]: sure

**Comment by himanshuc3** [comment]
I've added a mixin inplace of a media query. 
I apologize for the late commit. And yes, I do think I need to master my git skills ASAP.

**Comment by himanshuc3** [comment]
@catarak could you review this please. I think it might be done.

**Comment by himanshuc3** [comment]
how do you propose to remove it? By using flexbox?

**Comment by catarak** [comment]
yes!

**Comment by himanshuc3** [comment]
hey cassie. It was very disheartening for me to not qualify for GSOC as it was pretty important for me. Simultaneously, I realized that I could learn from this failure. Therefore, can you tell skills I were lacking so that I can improve upon them.

**Comment by himanshuc3** [comment]
Proposal on classroom workflow, definitely I did the wrong thing. But there were 2 proposals in by me. And honestly, I think I made some contributions to the community. So wasn't it to be based on merit basis rather than just submitting a proposal? I promise I wouldn't do such thing again. But is that the only reason?

**Comment by catarak** [comment]
@littlewonder this is not the place to call someone out. I ask that you refer to the [Code of Conduct](https://github.com/processing/p5.js-web-editor/blob/master/CODE_OF_CONDUCT.md) and treat others with respect. Because I believe your comment violates the Code of Conduct, I will be deleting it. I appreciate you bringing this issue up as I take plagiarism seriously, so feel free to reach out to me privately via email if you have any thoughts.

@himanshuc3 I have reached out to you via email. Please feel free to reply to me there.

**Pull request closed without merging**